nums/core/systems/systems.py

```diff
@@ -204,6 +220,24 @@ def num_cores_total(self) -> int:
         )
         return int(num_cores)
 
+    def register_actor(self, name: str, cls: type):
+        assert name not in self._actors
+        self._actors[name] = ray.remote(cls)
+
+    def make_actor(self, name: str, *args, device_id: DeviceID = None, **kwargs):
+        # Distribute actors round-robin over devices.
+        if device_id is None:
+            device_id = self._devices[self._actor_node_index]
+            self._actor_node_index = (self._actor_node_index + 1) % len(self._devices)
+        actor = self._actors[name]
+        node = self._device_to_node[device_id]
+        node_key = self._node_key(node)
+        options = {"resources": {node_key: 1.0 / 10 ** 4}}
```

Comment:
is 10**4 something specific or just a large number to accomodate many actors?

Reply:
Just a large number. I think working precision of resources is low (at the time I wrote this), which is why 10^4 is used and not something larger.